tests/Feature/HomePageTest.php

```diff
@@ -41,7 +41,7 @@ public function testHasInsertButton()
 
     public function testHasProducts()
     {
-        $products = factory(\App\Product::class, 10)->create();
+        $products = factory(\App\Product::class, 10)->states('admin')->create();
```

Comment:
Leave product alone, add a `$user` above it that looks like the one in `testHasOrders`